helpers/generic_helpers.py

```diff
@@ -139,7 +140,13 @@ def trim_video(config: Configuration, video_uri: str):
 
         # download
         with open(FFMPEG_BUFFER, "wb") as f:
-            f.write(get_blob(video_uri).download_as_string(client=None))
+            blob = get_blob(video_uri)
+            if blob:
+                f.write(blob.download_as_string(client=None))
+            else:
+                msg = f"Video URI: {video_uri} does not exist. Skipping execution."
+                logging.error(msg)
+                raise Exception(msg)
```

Comment:
Since Exception has a broad meaning, I think it is better to use the built-in errors if possible. This case is good for "ValueError" and value means video uri. What do you think? 
https://docs.python.org/3/library/exceptions.html#ValueError

This is not required, so if you have a reason to use "Exceptions", it's okay and please ignore my comment.

Reply:
Thanks for the comments Sho, addressed all of them in my second commit. Could you please approve?

Thanks!